docs/source/experiments/regression-robust04.md

```diff
@@ -0,0 +1,80 @@
+# PISA: Regressions for [Disks 4 & 5](https://trec.nist.gov/data_disks.html) (Robust04)
```

Comment:
It sounds like we something regressed for Disks 4 & 5. Maybe call it Regression Test(s) or something similar.